docs/examples/ad_demo.py

```diff
@@ -25,7 +25,7 @@
 
 # Create v2 devices
 with DeviceCollector():
-    det3 = areadetector.HDFStreamerDet(pv_prefix)
+    det3 = areadetector.Pilatus(pv_prefix)
 
 
 # And a plan
```

Comment:
This is probably wrong; we shouldn't be using StandardDetectors in flyscans

Reply:
We should probably create a `class DemoSimDetector(StandardDetector)` here to demonstrate how to do areaDetector, then use it in step scans. For the fly scan this plan is now wrong, and should be replaced by something that constructs a `HardwareTriggeredFlyable` then calls `fly_and_collect` on it. This is probably something for another PR so feel free to delete this file in this one